applications/rag/workloads.tfvars

```diff
@@ -18,6 +18,9 @@ project_id = "<project_id>"
 cluster_name     = "<cluster_name>"
 cluster_location = "us-central1"
 
+## CloudSQL variables
+private_network  = "ml-network"
```

Comment:
who creates this? is it during cluster creation? @umeshkumhar is there a better way to do this

Reply:
README needs update for how to use this variable.

Reply:
This is in the infra, so whatever you create there you use for Ray, CloudSQL and the Frontend right now.